### PR TITLE
Custom password validator.

### PR DESCRIPTION
**Pull request by AlecsGherghel**
### What does it fix?

Closes https://github.com/code4romania/ro-help/issues/142

### How has it been tested?

To test this locally you'll have to change `dev.py` settings file and make sure that `AUTH_PASSWORD_VALIDATORS` is not an empty list but something like:

```
AUTH_PASSWORD_VALIDATORS = {"NAME": "hub.password_validation.PasswordDifferentFromPrevious"}
```

Then, from the admin try to change the password of a user to the same password (as in, attempting to re-set the same password). It should throw a validation error as requested in the Github issue.

### CODE REVIEW

File: ro_help/hub/password_validation.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from django.utils.translation import ugettext_lazy as _
+from django.core.exceptions import ValidationError
```
Comment (vtemian) [inline ro_help/hub/password_validation.py]: Pretty clean!

Can you please just order the imports?

File: ro_help/hub/password_validation.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+from django.contrib.auth.hashers import check_password
+
+
+class PasswordDifferentFromPrevious:
+    def validate(self, password, user=None):
+        if not user:
+            return
+
+        # If the newly provided password is in fact still the old passoword
```
Comment (vtemian) [inline ro_help/hub/password_validation.py]:
```suggestion
        # If the newly provided password is in fact still the old password
```

File: ro_help/hub/password_validation.py (hunk 1 above)
Comment (AlecsGherghel) [reply ro_help/hub/password_validation.py]: Ofc. My bad. Regarding `.po` files i talked with @catileptic and for now maybe we can just mark strings for translation and when we're closer to the finish line we can generate (or re-generate) them for the entire project?

Comment (vtemian) [reply ro_help/hub/password_validation.py]: Sure! Let's generate them when we're almost done.

**Pull request merged**